src/components/Badges/badge.utils.ts

```
@@ -1,62 +1,109 @@
-// maps badge codes or '/badges/...' paths to static files under public/badges
+// per-badge metadata: asset path under public/badges, public-facing description,
+// and optional front-end display-name override. Backend codes/names stay the same;
+// overrides only affect what the user sees.
 import { PEANUTMAN_LOGO } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)